client/landing/jetpack-cloud/components/threat-item/index.tsx

```diff
@@ -12,6 +12,8 @@ import { Button } from '@automattic/components';
 import LogItem from '../log-item';
 import ThreatDescription from '../threat-description';
 import { Threat } from 'landing/jetpack-cloud/components/threat-item/types';
```

Comment:
We can tackle this in another PR when we have more clarity of the final shape of the threat object, but we should include those changes in the Threat type.